bin/telemetry_sim_LSSTCam

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python3
+
+import time
+import argparse
+import HeaderService.hutils as hutils
+import HeaderService.hscalc as hscalc
+import random
+import lsst.ts.salobj as salobj
+import lsst.ts.utils as ts_utils
+import asyncio
+
+
+def cmdline():
+
+    # Make conf_parser that hold values from a config.ini file
+    parser = argparse.ArgumentParser(description="Send telemetry to Header Client")
+
+    # The command-line arguments
+    parser.add_argument("--ra", action="store", default=None, type=float,
+                        help="The RA of the visit")
+    parser.add_argument("--dec", action="store", default=None, type=float,
+                        help="The DEC of the visit")
+    parser.add_argument("--rotpa", action="store", default=0.0, type=float,
+                        help="The ROTPA for the visit")
+
+    # Control imageName
+    parser.add_argument("--camcode", action="store", default='MT',
+                        help="The code for the telecope")
+    parser.add_argument("--controller", action="store", default='O',
+                        help="The controller (O for OCS, C for CCS)")
+    parser.add_argument("--dayobs", action="store", default=None,
+                        help="The observation day as defined by image name")
+    parser.add_argument("--seqnum", action="store", default=1, type=int,
+                        help="The sequence number from the image name")
+    parser.add_argument("--NSequence", action="store", default=3, type=int,
+                        help="Number of exposure in sequence")
+
+    parser.add_argument("--exptime", action="store", default=15, type=float,
+                        help="Exposure time in seconds")
+    parser.add_argument("--airmass", action="store", default=1.237, type=float,
+                        help="Airmass")
+    parser.add_argument("--ha", action="store", default='30.0', type=float,
+                        help="The HA")
+    parser.add_argument("--el", action="store", default=75.0, type=float,
+                        help="The elevation angle")
+    parser.add_argument("--az", action="store", default=45.0, type=float,
+                        help="The azimuth angle")
+
+    parser.add_argument("--m3port", action="store", default=1, type=int,
```

Comment:
Only necessary for AuxTel, can remove this.

Reply:
Removed